readme.md

```diff
@@ -18,10 +18,21 @@ files into the root of your SilverStripe project folder.
 
 **2**
 
+Open `dr.sh` in a text editor and update the following line to reference your theme folder.
```

Comment:
Not sure this needs to be here - not all projects use yarn, so it's not a necessary step. Could be moved down below though

Reply:
I think keeping it in a list of steps to follow is better, but maybe make the **2** say **2 (optional, for sites using Yarn)** or similar.

My reasoning for this is that I followed the instructions step by step, and then when I pushed it up, GitlabCI got an warning/error saying to change that line.

Alternatively, maybe there's a seperate set of steps for GitlabCI that we can put elsewhere?